COPYING

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2006-2020, Salvatore Sanfilippo
+Copyright (c) 2006-2021, Salvatore Sanfilippo
```

Comment:
Salvatore didn't add or change any code during 2021.

Perhaps we should add a line like

    Copyright (c) 2014-2021, All the contributors to Redis

Reply:
@zuiderkwast Curious to know as to why the start year was changed from 2006 to 2014?

Reply:
In the beginning there was only Salvatore. Later others started contributing. I'm not sure which year the more substantial contributions from others started. 2014 was just a guess. Maybe we should mention some more names. I didn't check git history.

Reply:
I'm not a lawyer, but I'm not sure that "all the" is a legal entity that can retain any rights. 
Also, I'm not sure what's the point of these lines, but I'm afraid to open the door to people starting to update the copyright year in all other source code files in the repo...

Reply:
I'm not a lawyer either, but I believe courts can look at the git history if they need to. It's a pity that these lines are in every file. Do you have some law person at RedisLabs?

OpenSSL writes "The OpenSSL Project Authors" in every file, e.g. https://github.com/openssl/openssl/blob/master/ssl/ssl_lib.c and they have an AUTHORS.md file listing all the names.

Reply:
If you guys are willing to trust AWS lawyers, we have several that focus on OSS I'll reach out to for guidance.